### DISCUSSION TITLE
Support for vertical tabs (like in Edge)

### DISCUSSION THREAD

**Discussion by umarhamzy** [Alex313031/Thorium-Win#86] [Ideas]
Here after Chris Titus' review. I'm currently installing the browser. However I don't see vertical tabs mentioned anywhere. Would absolutely love having support for vertical tabs since I have a lot of tabs open and accessing them in vertical tabs is so much more convenient for me.

**Comment by gz83** [comment]
There have been multiple duplicates or similar issues and may consider adding this feature in the future, the final decision is up to Alex.

@umarhamzy

**Reply by umarhamzy** [reply]
Looking forward!
